### DISCUSSION TITLE
安装在二级目录无法登录管理后台

### DISCUSSION THREAD

**Discussion by Nick838** [icret/EasyImages2.0#177] [教程]
如果程序安装在二级目录文件，例如: ./upload

无法登入后台进行设置，请修改文件: ./admin/index.php

找: action="/admin/index.php"

修改为: action=""

保存!重新上传文件

::后话!发觉程序对于安装在二级目录的测试环境不够严谨，有很多链接需要修正::

**Comment by icret** [comment]
原先开发的时候没有想过要放在二级目录，所以没有兼容。
如果需要放在二级目录，为何不直接使用一个二级域名绑定？

**Reply by Nick838** [reply]
主目录给其他程式霸佔了
现在 二级域名+二级目录
